development/frontEndApplication/chat/consumers.py

```diff
@@ -0,0 +1,169 @@
+                             norm='l2')
+
+# Fit and transform the vectorizer to get an score per word in an array returned
+#Vectorized_Data = vectorizer.fit_transform(DS1_data['Consumer complaint narrative'])
+Vectorized_Data = vectorizer.fit_transform(DS1_data['Consumer complaint narrative'].values.astype('U'))
+
+# Train the model on the training data
+
+
+# === Bard modeling ===
+#palm.configure(api_key='<api-Key>')
+#models = [m for m in palm.list_models() if 'embedText' in m.supported_generation_methods]
+
+#model = models[0]
+
+issue_to_int = {
+    "Trouble during payment process": 0,
+    "Struggling to pay mortgage": 1,
+    "Loan servicing, payments, escrow account": 2,
+    "Applying for a mortgage or refinancing an existing mortgage": 3,
+    "Loan modification,collection,foreclosure": 4,
+    "Closing on a mortgage": 5,
+    "Application, originator, mortgage broker": 6,
+    "Credit decision / Underwriting": 7,
+    "Incorrect information on your report": 8,
+    "Settlement process and costs": 9,
+    "Problem with a credit reporting company's investigation into an existing problem": 10,
+    "Improper use of your report": 11,
+    "Credit monitoring or identity theft protection services": 12,
+}
+
+# Create a new column called `Issue_Int`
+DS1_data["Issue_Int"] = DS1_data["Issue"].map(issue_to_int)
+
+
+def make_embed_text_fn(model):
+    def embed_fn(text: str) -> list[float]:
+        # Using the palm model, generate the embeddings for the text
+        return palm.generate_embeddings(model=model, text=text)['embedding']
+    return embed_fn
+
+# Creates the embedding of the given text
+def create_embeddings(model, text):
+    # Generate the embedding using the make_embed_text_fn function
+    embedding = make_embed_text_fn(model)(text)
+    # Return the embedding
+    return embedding
+
+# Creates the embedding for the train and the testdata
+
+class ChatConsumer(WebsocketConsumer):
+    def receive(self, text_data):
+        text_data_json = json.loads(text_data)
+        get_response.delay(self.channel_name, text_data_json)
+
+        async_to_sync(self.channel_layer.send)(
+            self.channel_name,
+            {
+                "type": "chat_message",
+                "text": {"msg": text_data_json["text"], "source": "user"},
+            },
+        )
+        rf_model.fit(Vectorized_Data, DS1_data['Issue'])  #
+        #rf_model.fit(Vectorized_Data, DS1_data['Embeddings'])  #
+
+        translation = translator.translate(text_data_json["text"])
+        score = vectorizer.transform([translation.text])
+        #DS1_data['Consumer complaint narrative'].iloc[0] = translation.text
+        #predictiondataset = create_embeddings(model, translation.text)
+        #print(predictiondataset)
+        #score = vectorizer.transform([predictiondataset])
+
+        prediction = rf_model.predict(list(score.toarray()))
+        #prediction = rf_model.predict(list((np.array(predictiondataset)).reshape(1, -1)))
```

Comment:
What I see in this code, you implemented an embedding based of bard, and the other one with tfifd. Are you using the random forest model with tfidf for predicting the issue? 
What does some of the commented code of the rf_model mean? If it is unnecessary, please remove it. Otherwise uncomment the commented code